frontend/src/app/messages/messages.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from '@angular/core';
+import { Http } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+
+import { environment } from '../../environments/environment';
+
+interface Message {
+  message: string;
+  type: string;
+}
+
+@Component({
+  selector: 'volontulo-messages',
+  templateUrl: './messages.component.html',
+})
+export class MessagesComponent {
+  messages$: Observable<Message[]>;
+  showMessages = true;
+
+  constructor(private http: Http) {
+    setTimeout(() => this.showMessages = false, 5000);
```

Comment:
I would set the timout time for a little bit more, as it disappeared quickly and I wasn't able to read it to the end. I would use 15-20 sec and add a dismiss button in a template.